src/sources/splunk_hec.rs

```diff
@@ -599,11 +615,13 @@ fn raw_event(
         .as_mut_log()
         .try_insert(event::log_schema().source_type_key(), "splunk_hec");
 
+    emit!(SplunkHECEventReceived);
+
     Ok(event)
 }
 
-#[derive(Debug, Snafu)]
-enum ApiError {
+#[derive(Clone, Copy, Debug, Snafu)]
+pub enum ApiError {
```

Comment:
If this is only used in the internal event, this can be limited to `pub(crate)`.

Reply:
Yep.